doc/contributors/design-proposals/db-migrations.md

```diff
@@ -0,0 +1,61 @@
+# Add database versioning and migrations to output database
```

Comment:
Super nit: I think the convention used everywhere else is `docs/...` not `doc/...` and we made a mistake in OLM repo.

Reply:
great point. updated.